mypyc/test-data/irbuild-bytes.test

```diff
@@ -62,14 +62,27 @@ L0:
     c = r6
     return 1
 
-
-[case testBytesJoin]
+[case testBytesOps]
```

Comment:
It would be better to have a separate test case for `len()`, since irbuild tests are fast to run. Small test cases are generally easier to maintain.

It's important to not have too many small run tests, however, since they have a lot of fixed overhead.

Reply:
Done!